src/handle_wrap.cc

```diff
@@ -37,6 +38,15 @@ void HandleWrap::Unref(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void HandleWrap::IsRefed(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  HandleWrap* wrap = Unwrap<HandleWrap>(args.Holder());
```

Comment:
Side note: This is unsafe, but is currently being done elsewhere in core. Am going to investigate how to not do this in the future.